bot/cogs/trivia.py

```diff
@@ -0,0 +1,449 @@
+import discord
+from discord.ext import commands
+import os
+import json
+import urllib.request
+import random
+
+correct = 'wrong'
+difficulty = 'easy'
+category = 'null'
+answers = []
+difficult = 'null'
+tokenID= '1234'
+
+amounts = {}
+userAnswers = {}
+
+
+class TriviaCog(commands.Cog):
```

Comment:
answer is never checked when running.
![image](https://user-images.githubusercontent.com/39353605/94882297-71d03d00-041c-11eb-9ed9-fe785b2fbac7.png)

Reply:
That's something I'll have to do some more looking on originally I used a -answer command but will add in a listener for those to not need a command